files/en-us/web/api/webgl2renderingcontext/getbuffersubdata/index.html

```diff
@@ -47,13 +47,16 @@ <h3 id="Parameters">Parameters</h3>
   <dd>A {{domxref("GLintptr")}} specifying the byte offset from which to start reading
     from the buffer.</dd>
   <dt><code>dstData</code></dt>
-  <dd>An {{jsxref("ArrayBuffer")}} or {{jsxref("SharedArrayBuffer")}} to which to write
-    the buffer data.</dd>
-  <dt>srcOffset {{optional_inline}}</dt>
-  <dd>A {{domxref("GLuint")}} specifying the element index offset where to start reading
+  <dd>An {{jsxref("ArrayBufferView")}} to copy the data to. If <code>dstData</code> is a
```

Comment:
```suggestion
  <dd>An {{domxref("ArrayBufferView")}} to copy the data to. If <code>dstData</code> is a
```
The macro `jsxref` is for features defined in JavaScript, like `ArrayBuffer`. `ArrayBufferView` is not defined in JavaScript but in the Web API (inside webidl), so, on MDN, we need to use the `domxref` macro.

This is one of the least obvious cases.

Reply:
Oh good catch. Thanks.